coalib/parsing/CliParsing.py

```diff
@@ -119,4 +119,10 @@ def check_conflicts(sections):
             ArgumentParser().error(
                 "'output' or 'relpath' cannot be used without `--json`.")
 
+        if (
+                section.get('non_interactive', False) and
```

Comment:
This empty line seems a bit strange.... Would it be possible to remove the empty newline?